source/shared/cpp/ObjectModel/BaseCardElement.cpp

```diff
@@ -27,11 +27,12 @@ BaseCardElement::BaseCardElement(CardElementType type) :
 
 void BaseCardElement::PopulateKnownPropertiesSet()
 {
-    m_knownProperties.insert({AdaptiveCardSchemaKeyToString(AdaptiveCardSchemaKey::Height),
-                              AdaptiveCardSchemaKeyToString(AdaptiveCardSchemaKey::IsVisible),
-                              AdaptiveCardSchemaKeyToString(AdaptiveCardSchemaKey::MinHeight),
-                              AdaptiveCardSchemaKeyToString(AdaptiveCardSchemaKey::Separator),
-                              AdaptiveCardSchemaKeyToString(AdaptiveCardSchemaKey::Spacing)});
+    m_knownProperties.insert(
```

Comment:
nit: max line width has changed from the old config? i have now problems with the changes, but i'm wondering if this's agreed.

Reply:
It's based on Paul's comments on /.clangformat, and achieved a balance between `BasedOnStyle: Microsoft` and the original `source/shared/cpp/.clang-format`